skrf/vectorFitting.py

```diff
@@ -165,14 +158,15 @@ def vector_fit(self, n_poles_real: int = 2, n_poles_cmplx: int = 2, init_pole_sp
         but they can also introduce unwanted resonances at frequencies well outside the fit interval.
         """
 
+        timer_start = timer()
+
```

Comment:
I though the timer was for debug and benchmark. Do you want to keep it in the merged code?

Reply:
It is still there on purpose. I thought it might be useful for other users to check the elapsed time. It can be another helpful criterion to select a good fitting setup.
I added a docstring for this attribute in https://github.com/scikit-rf/scikit-rf/pull/559/commits/981c894f2cd56e38f798417659b22571f9676acb, so it will be mentioned in the documentation.